### ISSUE TITLE
How does Vocabularies: RC/DRC counts appear? 

### ISSUE THREAD

**Issue by gywlsdms123** [OHDSI/WebAPI#2047]
Reorganized by modifying concept_id of CDM. (Changed the SNOMED voca of the condition table to KCD7.)

![image](https://user-images.githubusercontent.com/51704606/175002332-29b34902-3ab3-424d-b267-57ee0f12cba5.png)
But why is the **0** showing up in RC?

Is it because it was before I build Achilles?

Based on the information in the my current condition occurrence table, a non-zero number should show in RC.

**Comment by anthonysena** [comment]
> Reorganized by modifying concept_id of CDM. (Changed the SNOMED voca of the condition table to KCD7.)

Could you expand upon what you did here? You should not change the concept_ids in the vocabulary otherwise you will be completely out of sync with the OHDSI vocabulary.

> But why is the 0 showing up in RC?
> 
> Is it because it was before I build Achilles?
> 
> Based on the information in the my current condition occurrence table, a non-zero number should show in RC.

This is hard to answer - a few possible reasons: if you loaded your Achilles results after starting WebAPI, the results were probably not cached. I'd suggest restarting WebAPI to see if that fixes the RC display. If not, answering the 1st question may help us to understand why this is not working for you.

**Comment by gywlsdms123** [comment]
Let me tell you my current situation. 
1. Our CDM was mapped with snomed. However, we had to change this to KCD7, so we updated condition_concept_id to concept corresponding to kcd7. 
2. After completing the update, i am currently running Achilles. 
3. When I restarted WebAPI, RC showed **timeout**.

Data from the Domains, Vocabulary options, and concept tables are immediately applied to the Atlas search page. 
However, considering that only count is not applied, I think it is related to Achilles. Is it because I'm still running Achilles?
